python/pyspark/mllib/classification.py

```diff
@@ -50,6 +51,15 @@ def setThreshold(self, value):
         """
         self._threshold = value
 
+    def getThreshold(self):
```

Comment:
I realized that this should probably be a property called "threshold" instead of a method like this, in order to fit with existing Python conventions.